osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/config/JasperWirelessConfig.java

```diff
@@ -63,6 +63,9 @@
   @Value("${jwcc.api_version}")
   private String apiVersion;
 
+  @Value("${jwcc.validity_period:0}")
```

Comment:
Is it clear what the time unit is for this? ms, seconds, hours? Maybe make it part of the property name.